.pre-commit-config.yaml

```diff
@@ -7,8 +7,9 @@ repos:
           - "--unsafe"
       - id: end-of-file-fixer
       - id: trailing-whitespace
+        exclude: "README.md"
   - repo: https://github.com/psf/black
```

Comment:
Note that black now has a specialist repo for pre-commit at https://github.com/psf/black-pre-commit-mirror